index.bs

```diff
@@ -84,7 +84,7 @@ The ABNF (Augmented Backus-Naur Form) syntax for the `Device-Memory` header fiel
 <h3 dfn>Computing Device Memory Value</h3>
 The value is calculated by using the actual device memory in MiB then rounding it to the nearest number where only the most signicant bit can be set and the rest are zeros (nearest power of two). Then dividing that number by 1024.0 to get the value in GiB.
 
-An upper bound and a lower bound should be set on the list of values.
+An upper bound and a lower bound SHOULD be set on the list of values.
```

Comment:
Uppercasing RFC2119 keywords isn't strictly necessary; per RFC8174's clarification

(If you wanted, we could also include a section like the one in INFRA: https://infra.spec.whatwg.org/#conformance), but I don't think it's required.

Reply:
Lowercased